src/app/(auth)/sign-up/page.tsx

```
@@ -1,8 +1,18 @@
+import { headers } from "next/headers";
+const Page = async () => {
+  const session = await auth.api.getSession({
+    //deconstrcution of head parameters
+    headers: await headers(),
+  });
 
-const Page=()=>{
-    
-    return <SignUpView/>
-}
+  if (!!session) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)